lr_face/models.py

```diff
@@ -19,7 +19,7 @@ def fit(self, X, y):
 
     def predict_proba(self, X, ids=None):
         # assert X.shape[2:4] == self.resolution
-        if X.shape[1] != 2:
+        if np.array(X).shape[1] != 2:
```

Comment:
Is `X` hier nog geen numpy array? Zo niet, dan gaat denk ik die `ValueError` hieronder nog fout, want daar wordt nog `X.shape` gebruikt

Reply:
true